ops/docker/Dockerfile.deployer

```diff
@@ -34,4 +34,4 @@ COPY packages/contracts/test/helpers/constants.ts ./test/helpers/constants.ts
 COPY packages/contracts/scripts ./scripts
 
 COPY ./ops/scripts/deployer.sh .
-ENTRYPOINT yarn run deploy
+CMD ./ops/scripts/deployer.sh
```

Comment:
It should be fine to skip using `yarn` for the deployer, but other services may consume env vars injected into the runtime from `yarn`